incubator/cni-genie/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "cni-genie.fullname" . }}
+  labels:
+    app: {{ template "cni-genie.name" . }}
+    chart: {{ template "cni-genie.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
Can you please use the newer format for the [labels in the docs](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata). These come from a common set documented by Kubernetes. This applies all over.